src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let counter = 0;
+  let finalSum;
+
+  const infiniteAdder = (...args) => {
```

Comment:
Is it gonna be more than one argument? 
So if not - let function takes only one argument